app/assets/stylesheets/darkswarm/modals.css.scss

```diff
@@ -20,9 +20,10 @@ dialog, .reveal-modal {
   // Small - when modal IS full screen
   @media only screen and (max-width: 640px) {
     height: 500px;
```

Comment:
Hey, now that we are on this one.

I see @HugsDaniel  change past Feb2018:
from
-    min-height: 100%; // This is needed to make the height not the height of whole content page
to
+    height: 500px;

Is there a reason why the modals (for example the login modal) should not take the full screen if there's stuff to be shown?
I'd go for max-height: 100% here and use the empty space below the 500px point.
What do you think?

Reply:
I'm not sure, but I'm thinking the `min-height: 100%` could have been made before the following was added:

```css
body.modal-open {
  overflow: hidden;                                                                                                                                                                                                                                                        
}
```

The comment would have been true - 100% height for the modal would have occupied the full length of the then scrollable `body`.

Reply:
@luisramos0 I removed `height: 500px` for small screens, to allow the modal to go past this height. There doesn't seem to be any issue from doing that.